scenarioo-server/src/main/java/org/scenarioo/dao/search/elasticsearch/ElasticSearchAdapter.java

```diff
@@ -72,10 +74,11 @@ public ElasticSearchAdapter() {
 			int portSeparator = endpoint.lastIndexOf(':');
 			String host = endpoint.substring(0, portSeparator);
 			int port = Integer.parseInt(endpoint.substring(portSeparator + 1), 10);
-			client = TransportClient.builder().build()
-					.addTransportAddress(new InetSocketTransportAddress(InetAddress.getByName(host), port));
+			client = new PreBuiltTransportClient(Settings.builder()
+				.put("cluster.name", "elasticsearch").build())
```

Comment:
What is this clustername for? Is it relevant to make this configurable? Or could we maybe just use something a little bit less generic, something like "scenarioo-elasticsearch" or what are the implications of this?

Reply:
The clustername was needed for the connection to work. We should in some way make this configurable. But firstly we wanted to get it running.

Reply:
yes, sure -- hmm, seems we need to make it configurable and document it.